docs/reST/ref/font.rst

```diff
@@ -472,6 +472,20 @@ solves no longer exists, it will likely be removed in the future.
 
       .. ## Font.get_descent ##
 
+   .. method:: set_pointsize
+
+      | :sl:`set the point size of the font`
+      | :sg:`set_pointsize(bool) -> None`
+
```

Comment:
Not sure if intentional, but there is no description/body here

Reply:
I felt like the small description was enough, but I can add more to it if needed.